packages/vite/src/node/server/middlewares/static.ts

```diff
@@ -49,7 +49,6 @@ export function serveStaticMiddleware(
     if (path.extname(cleanUrl(url)) === '.html') {
       return next()
     }
-
```

Comment:
**nitpick:** IMO removing this blank line doesn't have anything to do with the PR 🤔 
I think it was there for better readability